### PR TITLE
Disabled listing fix

### PR DESCRIPTION
**Pull request by flesher1**
Issue https://github.com/aml-development/ozp-center/issues/868 

The wording is what I made, please review. The button color and text is also not final.

There is a different warning for an admin and for a regular user. A user will get an error and a admin will get a warning and the listing will remain open. Please open a disables listing as both a regular user and admin to test fully.

### CODE REVIEW

File: app/js/components/quickview/index.jsx [hunk 1]
```diff
@@ -82,6 +83,19 @@ var Quickview = React.createClass({
         var { shown, listing } = this.state;
         var ActiveRouteHandler = this.getActiveRouteHandler();
         var owners, tabs;
+
+        if (listing){
+            if ((!currentUser.isAdmin()) && (listing.isEnabled == false)){
+                //if (listing.isEnabled == false){
```
Comment (blynch11p) [inline app/js/components/quickview/index.jsx]: This line and line 93 can be removed.

**Comment by blynch11p** [comment]
Tested:
Regular users see a notice that the listing is no longer available
'Admin' users are notified that the listing is disabled for regular users, but the admin can still view the listing

But...
Any user (org steward/listing owner) that can enable/disable a listing, should also see the notice that an Admin sees.  Meaning, any user type for a Listing that can see the Admin tab, should not be blocked from seeing the listing.  [this was discussed/confirmed with @rkenyon1969]

**Comment by blynch11p** [comment]
I like it.

Deleted, for all users, does not show listing, displays dialog saying listing deleted

Disabled:
Regular users see a notice that the listing is no longer available
Admin/Org Steward/Owner users are notified that the listing is disabled for regular users, but the admin can still view the listing

👍

**Comment by rkenyon1969** [comment]
1. pulled branch
2. reviewed code
3. tested

:+1:

**Pull request merged**